src/sentry/api/serializers/models/organization_member.py

```diff
@@ -191,3 +191,27 @@ def serialize(
         d = super().serialize(obj, attrs, user)
         d["projects"] = attrs.get("projects", [])
         return d
+
+
+class OrganizationMemberSCIMSerializer(Serializer):  # type: ignore
+    def __init__(self, expand: Optional[Sequence[str]] = None) -> None:
+        self.expand = expand or []
+
+    def serialize(
+        self, obj: OrganizationMember, attrs: Mapping[str, Any], user: Any, **kwargs: Any
+    ) -> MutableMapping[str, JSONData]:
+        from sentry.scim.endpoints.utils import SCIM_SCHEMA_USER  # type: ignore
```

Comment:
Is this a late import to avoid cycles?

Reply:
yes -- wondering if i should move this serializer to the scim code to avoid that. could also create a scim constants file and move all those schemas to it. leaning towards a constants file, feels nice to keep it with its serializer siblings here.